controllers/feeds/getFeedsV2.js

```diff
@@ -91,7 +91,7 @@ const isValidActivity = async (item, conditions) => {
 
   // filter by threshold
   const threshold = ACTIVITY_THRESHOLD[feed.toUpperCase()];
-  return (item.final_score || 0) >= threshold;
+  return (item.final_score || 0) >= (threshold || 0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for checking the activity threshold has been changed to handle cases where `threshold` is undefined. This is a good change as it prevents potential errors when `ACTIVITY_THRESHOLD[feed.toUpperCase()]` is not defined. However, it might be better to handle this case explicitly and log an error or throw an exception if the threshold for a feed is not defined. This would make it easier to debug issues related to missing thresholds.
```diff
- return (item.final_score || 0) >= (threshold || 0);
+ if (threshold === undefined) {
+   console.error(`Threshold not defined for feed: ${feed}`);
+   throw new Error(`Threshold not defined for feed: ${feed}`);
+ }
+ return (item.final_score || 0) >= threshold;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->